src/filters/pages.cjs

```diff
@@ -319,6 +319,9 @@ const byYear = (collection) => {
     }));
 };
 
+const addCallToAction = (pageURL) =>
+  _.isString(pageURL) && pageURL.match(/^(\/work\/|\/services\/)/) !== null;
```

Comment:
This works for me! Might be slightly more readable using `.startswith()`, but 🤷

Reply:
Oh good point! I will switch it over so that is more readable